src/components/Navigator/NavGroup/NavGroup.tsx

```diff
@@ -0,0 +1,106 @@
+/* External dependencies */
+import React, { useCallback, memo } from 'react'
+import { noop, isNil } from 'lodash-es'
+
+/* Internal dependencies */
+import { Typography } from 'Foundation'
+import { Icon, IconSize, isIconName } from 'Components/Icon'
+import type NavGroupProps from './NavGroup.types'
+import {
+  Item,
+  Wrapper,
+  ChildrenWrapper,
+  LeftIconWrapper,
+  ChevronWrapper,
+  ContentWrapper,
+  RightContentWrapper,
+} from './NavGroup.styled'
+
+export const NAV_GROUP_TEST_ID = 'bezier-react-nav-group'
+
+function NavGroup({
+  as,
+  testId = NAV_GROUP_TEST_ID,
+  name,
+  style,
+  className,
+  interpolation,
+  children,
+  content,
+  rightContent,
+  leftIcon,
+  leftIconColor = 'txt-black-dark',
+  open,
+  onClick = noop,
+}: NavGroupProps) {
+  const handleClickItem = useCallback((e?: React.MouseEvent) => {
+    onClick(e, name)
+  }, [
+    name,
+    onClick,
+  ])
+
+  const hasChildren = !isNil(children)
+  const chevronIconName = open ? 'chevron-small-down' : 'chevron-small-right'
+  const showLeftIcon = isIconName(leftIcon)
+
+  return (
+    <Wrapper
+      role="menuitem"
+      aria-haspopup={hasChildren}
+      aria-expanded={open}
+      aria-controls={name}
+    >
+      <Item
+        as={as}
+        open={open}
+        style={style}
+        className={className}
+        interpolation={interpolation}
+        onClick={handleClickItem}
+        data-testid={testId}
+      >
```

Comment:
```suggestion
    <Wrapper role="none">
      <Item
        as={as}
        open={open}
        style={style}
        className={className}
        interpolation={interpolation}
        onClick={handleClickItem}
        data-testid={testId}
        role="menuitem"
        aria-haspopup={hasChildren}
        aria-expanded={open}
        aria-controls={name}
      >
```

키보드 포커싱이 가게 하려면 `Item` 컴포넌트가 `a` 혹은 `button` 엘리먼트가 되어야 할 거 같아요! 
[W3 - Navigation Menubar Example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html#)를 참고했어요. 둘 중 뭐가 더 어울리는지는 저도 공부를 좀 해봐야될 거 같아요.

저번에 공유했던 사이트 같긴 한데, [이 사이트](https://nuli.navercorp.com/community/article/1132961) 설명이 잘 되어있네요

Reply:
키보드 컨트롤은 tabindex를 통해 기본적으로 포커스를 받지 않는 태그에도 포커스가 가능하도록 설정할 수 있지만, 구조의 통일성을 위해서 li 안에 a나 button 같은 태그를 추가하는것도 나쁘지 않아보입니다

Reply:
`a` 는 페이지 이동이 있을 경우에 사용하는게 좋고, href를 추가하는게 권장되는데요, 메뉴를 펼치기/접기만 하는 navGroup에 `a`는 페이지가 다른 곳으로 이동할 수 있기 때문에 `button` 사용이 좋아 보여요. Storybook도 페이지 이동이 있는 하위 메뉴에만 `a`를 쓰고 상위에는 `button`을 쓰고 있네요!

Reply:
오 좋네요 👍